save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DemoManagerController.kt

```diff
@@ -53,30 +63,48 @@ class DemoManagerController(
     fun addDemo(
         @PathVariable organizationName: String,
         @PathVariable projectName: String,
-        @RequestBody demoToolRequest: NewDemoToolRequest,
+        @RequestBody demoDto: DemoDto,
         authentication: Authentication,
-    ): Mono<Unit> = demoToolRequest.toMono()
+    ): Mono<EmptyResponse> = blockingToMono {
+        projectService.findByNameAndOrganizationNameAndStatusIn(
```

Comment:
what's about `findByNameAndOrganizationNameAndCreatedStatus`

Reply:
Oh sure, will fix it

Reply:
Done